src/accel/LocalTransporter.cc

```diff
@@ -200,6 +203,35 @@ void LocalTransporter::Finalize()
     CELER_ENSURE(!*this);
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * Get the accumulated action times.
+ */
+auto LocalTransporter::GetActionTime() const -> MapStrReal
+{
+    CELER_EXPECT(*this);
+
+    MapStrReal result;
+    auto const& action_seq = step_->actions();
+    if (num_streams_ == 1
+        && (action_seq.sync()
+            || !celeritas::getenv("CELER_DISABLE_DEVICE").empty()))
```

Comment:
If I'm following the triple negative: can't this be
```suggestion
            || !celeritas::device()))
```
? The device should be set if and only if we're running the celeritas offload on device.

Reply:
Yep that should work, much easier to read.